### PR TITLE
Migrate documentation for the newly added props.

### PR DESCRIPTION
**Pull request by ritikamotwani**

### CODE REVIEW

File: src/Display/Slider/Slider.stories.tsx [hunk 1]
```diff
@@ -67,3 +80,59 @@ RemoveDots.args = { removeDots: true };
 
 export const InitialItem = Template.bind({});
 InitialItem.args = { initialItem: 2 };
+
+export const CustomizedRenderLeftIcon: Story<Props> = () => (
```
Comment (westwood846) [inline src/Display/Slider/Slider.stories.tsx:84]: Can't you do
```
export const InitialItem = Template.bind({});
CustomizedRenderLeftIcon.args = { 
    renderLeftIcon={(disabled: boolean) => {
      const color = disabled ? Greyscale.grey : SecondaryColor.actionblue;
      return <ArrowBackDoubleIcon color={color} />;
    }}
};
```
here?

File: src/Display/Slider/Slider.tsx [hunk 2]
```diff
@@ -217,7 +217,9 @@ export interface Props {
   arrowWhite?: boolean;
   /** If <code>true</code>, dot controls will be removed */
   removeDots?: boolean;
+  /** Sets the left icon. Receives the "disabled" value as an argument. */
```
Comment (westwood846) [inline src/Display/Slider/Slider.tsx]: >value
Might be slightly more helpful to call it "disabled flag"

**Review by westwood846 — COMMENTED** [review]
Left a question and a comment.

File: src/Display/Slider/Slider.stories.tsx (hunk 1 above)
Comment (ritikamotwani) [reply src/Display/Slider/Slider.stories.tsx:84]: I had done this initially but this function wasn't visible in the source code 
The source code becomes like
<Slider renderLeftIcon={() => {}}>
..........
</Slider>

Comment (westwood846) [reply src/Display/Slider/Slider.stories.tsx:84]: Ooh I see, then this is okay.

**Review by westwood846 — APPROVED** [review]

**Pull request merged**